FEM/rf_mmp_new.h

```diff
@@ -311,6 +317,60 @@ class CMediumProperties
 	double graindiameter;
 	double hydraulicrad;
 	double betaexpo;
+    // HS: borehole heat exchanger related parameters
```

Comment:
e.g., a class similar to class CMediumProperties to read data, hold data and compute on the data.